test/unit/index.test.ts

```diff
@@ -0,0 +1,141 @@
+  'Binary',
+  'BSON',
+  'BSONRegExp',
+  'BSONSymbol',
+  'BSONType',
+  'CancellationToken',
+  'ChangeStream',
+  'ChangeStreamCursor',
+  'ClientSession',
+  'Code',
+  'Collection',
+  'CommandFailedEvent',
+  'CommandStartedEvent',
+  'CommandSucceededEvent',
+  'Compressor',
+  'ConnectionCheckedInEvent',
+  'ConnectionCheckedOutEvent',
+  'ConnectionCheckOutFailedEvent',
+  'ConnectionCheckOutStartedEvent',
+  'ConnectionClosedEvent',
+  'ConnectionCreatedEvent',
+  'ConnectionPoolClearedEvent',
+  'ConnectionPoolClosedEvent',
+  'ConnectionPoolCreatedEvent',
+  'ConnectionPoolMonitoringEvent',
+  'ConnectionReadyEvent',
+  'CURSOR_FLAGS',
+  'Db',
+  'DBRef',
+  'Decimal128',
+  'Double',
+  'ExplainVerbosity',
+  'FindCursor',
+  'GridFSBucket',
+  'GridFSBucketReadStream',
+  'GridFSBucketWriteStream',
+  'GSSAPICanonicalizationValue',
+  'Int32',
+  'ListCollectionsCursor',
+  'ListIndexesCursor',
+  'Logger',
+  'LoggerLevel',
+  'Long',
+  'Map',
+  'MaxKey',
+  'MinKey',
+  'MongoAPIError',
+  'MongoAWSError',
+  'MongoBatchReExecutionError',
+  'MongoBulkWriteError',
+  'MongoChangeStreamError',
+  'MongoClient',
+  'MongoCompatibilityError',
+  'MongoCursorExhaustedError',
+  'MongoCursorInUseError',
+  'MongoDecompressionError',
+  'MongoDriverError',
+  'MongoError',
+  'MongoErrorLabel',
+  'MongoExpiredSessionError',
+  'MongoGridFSChunkError',
+  'MongoGridFSStreamError',
+  'MongoInvalidArgumentError',
+  'MongoKerberosError',
+  'MongoMissingCredentialsError',
+  'MongoMissingDependencyError',
+  'MongoNetworkError',
+  'MongoNetworkTimeoutError',
+  'MongoNotConnectedError',
+  'MongoParseError',
+  'MongoRuntimeError',
+  'MongoServerClosedError',
+  'MongoServerError',
+  'MongoServerSelectionError',
+  'MongoSystemError',
+  'MongoTailableCursorError',
+  'MongoTopologyClosedError',
+  'MongoTransactionError',
+  'MongoUnexpectedServerResponseError',
+  'MongoWriteConcernError',
+  'ObjectId',
+  'ObjectID',
+  'OrderedBulkOperation',
+  'ProfilingLevel',
+  'Promise',
+  'ReadConcern',
+  'ReadConcernLevel',
+  'ReadPreference',
+  'ReadPreferenceMode',
+  'ReturnDocument',
+  'ServerApiVersion',
+  'ServerClosedEvent',
+  'ServerDescriptionChangedEvent',
+  'ServerHeartbeatFailedEvent',
+  'ServerHeartbeatStartedEvent',
+  'ServerHeartbeatSucceededEvent',
+  'ServerOpeningEvent',
+  'ServerType',
+  'SrvPollingEvent',
+  'Timestamp',
+  'TopologyClosedEvent',
+  'TopologyDescriptionChangedEvent',
+  'TopologyOpeningEvent',
+  'TopologyType',
+  'UnorderedBulkOperation',
+  'WriteConcern'
+];
+
+describe('mongodb entrypoint', () => {
+  it('should export all and only the expected keys in expected_exports', () => {
+    expect(sorted(Object.keys(mongodb), byStrings)).to.deep.equal(
+      sorted(EXPECTED_EXPORTS, byStrings)
+    );
+  });
+
+  it('should export keys added by ts-node as undefined', () => {
+    expect(TS_NODE_EXPORTS).to.have.length.greaterThan(0); // Are there no longer fake ts-node exports?
```

Comment:
I don't think we need the comment unless it's going to suggest an action (e.g., delete the declared const array because the test is now an no-op)